src/main/java/models/CreatureCard.java

```diff
@@ -32,6 +33,17 @@ public void removeHp( int dmg){
 
         this.hp -= dmg;
     }
+    public void increaseIgnRoundCounter() {
+        ignRoundCounter+=1;
+    }
+
+    public int getIgnRoundCounter() {
+        return ignRoundCounter;
+    }
+
+    public void setIgnRoundCounter(int ignRoundCounter) {
```

Comment:
Instead of `setIgnRoundCounter` perhaps `resetIgniteCounter` that set ignCounter back to 0.